src/riak_kv_pb_timeseries.erl

```diff
@@ -127,6 +130,53 @@ process(#tsputreq{table = Bucket, columns = _Columns, rows = Rows}, State) ->
             {reply, missing_helper_module(Bucket, BucketProps), State}
     end;
 
+%% @ignore DELETE (as if)
+process(#tsdelreq{table = Table, key = CompoundKey_,
+                  vclock = VClock_, timeout = Timeout},
+        State) ->
+    Options =
```

Comment:
It's a remnant of a comprehensive set of options. Replaced with an `if` statement.